backend/logic/users/users.js

```diff
@@ -142,9 +142,11 @@ var getStudentData = async studentID => {
   let completedCourses, major, minors;
   try {
     completedCourses = await conn.query(
-      `SELECT course_code, semester
+    `SELECT course_offerings.course_code, course_offerings.semester, 
```

Comment:
minor suggestion, consider using a variable to store a long query for better readability.

Reply:
I attempted to indent it so it's still readable :')